assets/src/apps/scheduler/scheduler-slice.ts

```diff
@@ -233,7 +235,7 @@ const schedulerSlice = createSlice({
           datesChanged =
             datesChanged || action.payload.endDate.getTime() !== mutableItem.endDateTime?.getTime();
 
-          mutableItem.endDate = new DateWithoutTime();
+          mutableItem.endDate = new DateWithoutTime(2020, 1, 1);
```

Comment:
Do we want to update this to use the 3 param setFullYear as well?

Reply:
The DateWithoutTime doesn't have the 3-param version of setYear.